setEnvironmentVariables.ps1

```diff
@@ -13,7 +13,7 @@
 [Environment]::SetEnvironmentVariable("DATALAND_EDC_USER", $env:GITHUB_USER, [System.EnvironmentVariableTarget]::User)
 [Environment]::SetEnvironmentVariable("DATALAND_SKYMINDERCLIENT_TOKEN", $env:GITHUB_TOKEN, [System.EnvironmentVariableTarget]::User)
 [Environment]::SetEnvironmentVariable("DATALAND_SKYMINDERCLIENT_USER", $env:GITHUB_USER, [System.EnvironmentVariableTarget]::User)
-[Environment]::SetEnvironmentVariable("ELECTRON_EXTRA_LAUNCH_ARGS", "--ignore-connections-limit=preview-dataland.duckdns.org,dev-dataland.duckdns.org,dataland-local.duckdns.org", [System.EnvironmentVariableTarget]::User)
+[Environment]::SetEnvironmentVariable("ELECTRON_EXTRA_LAUNCH_ARGS", "--ignore-connections-limit=preview-dataland.duckdns.org,dev-dataland.duckdns.org,dev2-dataland.duckdns.org,dataland-local.duckdns.org", [System.EnvironmentVariableTarget]::User)
```

Comment:
Don't we just need "dataland-local.duckdns.org" here?

Reply:
Why? Is electron no longer launched in the CD/CI?

Reply:
Dann ja.

Reply:
Auch gut.